src/app/public/modules/vertical-tabset/fixtures/vertical-tabset.component.fixture.ts

```diff
@@ -20,6 +20,8 @@ export class VerticalTabsetTestComponent {
   public active: boolean = true;
   public tabDisabled: boolean = true;
 
+  public maintainTabContent: boolean = false;
```

Comment:
Check your alphabetical ordering here.

Reply:
Done. Also fixed legacy issues with that in this file.